src/model_outputfiles.cpp

```diff
@@ -9,36 +9,114 @@
 #include <fstream>
 #include <filesystem>
 
-///////////////////////////////
-// AUX FUNCTION DECLARATIONS //
-///////////////////////////////
-
-std::string timeNow();
-
 ///////////////////
 // RESULT REPORT //
 ///////////////////
 
+// TODO remove if completely obselete
 bool Model::createOutputFolder(std::string file_name){
   // folder name based on current time to avoid overwriting output files with successive calculations
   calc_time = timeNow();
   output_folder = "./output/" + file_name + " MoloVol/" + calc_time + "/";
+  // TODO create directories didn't seem to work on MacOS, need to fix this issue later
   if(std::filesystem::create_directories(output_folder)){
     return true;
   }
   else{
     output_folder = "./";
     return false;
   }
+  return true;
 }
 
-// TODO revamp report
-void Model::createReport(std::string input_filepath, std::vector<std::string> parameters){
-  std::ofstream output_report(output_folder+"/MoloVol result report.txt");
-  output_report << "MoloVol program: calculation results report\n\n";
-  output_report << "Structure file analyzed: " << input_filepath << "\n";
-  for(size_t i = 0; i < parameters.size(); i++){
-    output_report << parameters[i] << "\n";
+void Model::createReport(){
+  std::ofstream output_report(output_folder+"/MoloVol report "+calc_time+".txt");
+  output_report << "\n";
+  output_report << "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n\n";
+  output_report << "   MM           MM            LL           VV           VV           LL   \n";
+  output_report << "   MMM         MMM            LL            VV         VV            LL   \n";
+  output_report << "   MMMM       MMMM            LL             VV       VV             LL   \n";
+  output_report << "   MM MM     MM MM            LL              VV     VV              LL   \n";
+  output_report << "   MM  MM   MM  MM    OOOO    LL    OOOO       VV   VV       OOOO    LL   \n";
+  output_report << "   MM   MM MM   MM   OO  OO   LL   OO  OO       VV VV       OO  OO   LL   \n";
+  output_report << "   MM    MMM    MM   OO  OO   LL   OO  OO        VVV        OO  OO   LL   \n";
+  output_report << "   MM     M     MM    OOOO    LL    OOOO          V          OOOO    LL   \n\n";
+  output_report << "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>\n\n";
+  output_report << "Source code available at https://github.com/jmaglic/MoloVol under the MIT licence\n";
+  output_report << "Copyright � 2020-2021 Jasmin B. Maglic, Roy Lavendomme\n\n";

```

Comment:
getting a warning for this line: "illegal character encoding in string literal"

I may have to do something about that. however, the copyright symbol does show up correctly. we could consider using "(c)" instead

Reply:
We can use (c) but it is very strange for you to get such a warning since the copyright character is part of the extended ACSII table and I have no issue about it (no warning or error during compilation).